libs/custom-paper-handler/jest.config.js

```diff
@@ -0,0 +1,9 @@
+const shared = require('../../jest.config.shared');
+
+/**
+ * @type {import('@jest/types').Config.InitialOptions}
+ */
+module.exports = {
+  ...shared,
+  collectCoverageFrom: [...shared.collectCoverageFrom, '!src/cli/**/*.ts'],
```

Comment:
Coverage disabled in top-level jest config

Reply:
[b1c4ecc](https://github.com/votingworks/vxsuite/pull/3472/commits/b1c4ecc688e49e2047b2feaf2516f1429f630b22)